alerts/absent_alerts.libsonnet

```diff
@@ -38,6 +38,27 @@
           },
         ],
       },
+      {
+        name: 'ceph-mds-status',
+        rules: [
+          {
+            alert: 'CephMdsMissingReplicas',
+            expr: |||
+              sum(ceph_mds_metadata{%(cephExporterSelector)s} == 1) < 2
+            ||| % $._config,
+            'for': $._config.mdsMissingReplicasAlertTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Insufficient replicas for storage metadata service.',
+              description: 'Minimum required replicas for storage metadata service not available.',
```

Comment:
Message and Description still give the exact same information.

Reply:
Let me make it more meaningful.

Reply:
Also meaning has to be same ;)